<ui version="4.0" >
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="pix" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../data.qrc" >:/gui/pics/welcome.png</pixmap>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="5" >
    <widget class="QLabel" name="disclamer" >
     <property name="text" >
      <string>All software developed by TUBITAK/UEKAE is licensed under GNU/GPL. Other software have their corresponding licenses. To install and/or use Pardus, you must accept them. Do you accept GNU/GPL and other, corresponding licences?</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="3" >
    <layout class="QGridLayout" >
     <property name="spacing" >
      <number>10</number>
     </property>
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>12</number>
       </property>
       <item>
        <widget class="QPushButton" name="gplButton" >
         <property name="minimumSize" >
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text" >
          <string>Read GPL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rebootButton" >
         <property name="minimumSize" >
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text" >
          <string>Reboot System</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="styleSheet" >
        <string notr="true" >border:none;</string>
       </property>
       <property name="title" >
        <string/>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="accept" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text" >
           <string>Yes, I accept.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="not_accept" >
          <property name="focusPolicy" >
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text" >
           <string>No, I don't accept.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="5" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>accept</tabstop>
  <tabstop>not_accept</tabstop>
  <tabstop>gplButton</tabstop>
  <tabstop>rebootButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../data.qrc" />
 </resources>
 <connections/>
</ui>
